subt/local_planner.py

```diff
@@ -12,16 +12,17 @@ def normalize_angle(angle):
 
 
 class LocalPlannerRef:
-    def __init__(self, scan_right=math.radians(-135), scan_left=math.radians(135), direction_adherence=math.radians(90), max_obstacle_distance=1.5, obstacle_influence=1.2):
+    def __init__(self, scan_right=math.radians(-135), scan_left=math.radians(135), direction_adherence=math.radians(90), max_obstacle_distance=1.5, obstacle_influence=1.2, scan_subsample=1):
         self.last_scan = None
         self.scan_right = scan_right
         self.scan_left = scan_left
         self.direction_adherence = direction_adherence
         self.max_obstacle_distance = max_obstacle_distance
         self.obstacle_influence = obstacle_influence
+        self.scan_subsample = scan_subsample
 
     def update(self, scan):
-        self.last_scan = scan
+        self.last_scan = scan[::self.scan_subsample] if self.scan_subsample > 1 else scan
```

Comment:
We might consider to take minimum of the measurements we want to convert to just one. If we don't, we run into the risk of missing thin obstacles like poles or hanging cables. The lidar sensor with smaller resolution presumably have wider beem (?).

Reply:
As Jakub pointed in mail there is also opposite problem with noise point readings which could be filtered out say by median.

Reply:
Note, the sub-sampling we are talking here about is from 1/3 degrees to 1 degree, i.e. 
```
>>> math.sin(math.radians(1))
0.01745240643728351
```
approximately 2cm wide objects at 1m